gm4_pulverizers/data/pulverizers/functions/chance/set_slot.mcfunction

```diff
@@ -0,0 +1,12 @@
+#@s = pulverizers containing a recipe results
+#Run from chance/apply_chance
+
+execute if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"log"}}}}]} if score chance_stack gm4_stack_size matches 1.. run replaceitem block ~ ~ ~ container.7 stick{gm4_pulverizers:{chance:1}} 1
+execute if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"planks"}}}}]} if score chance_stack gm4_stack_size matches 1.. run replaceitem block ~ ~ ~ container.7 stick{gm4_pulverizers:{chance:1}} 1
+execute if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"blaze_rod"}}}}]} if score chance_stack gm4_stack_size matches 1.. run replaceitem block ~ ~ ~ container.7 blaze_powder{gm4_pulverizers:{chance:1}} 1
+execute if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"bone"}}}}]} if score chance_stack gm4_stack_size matches 1.. run replaceitem block ~ ~ ~ container.7 bone_meal{gm4_pulverizers:{chance:1}} 1
```

Comment:
All of these commands run even though `if score chance_stack gm4_stack_size matches 1..` might not pass. Move this check to `apply_chance` before this function call.

Additionally (like mentioned in `set_stack_count`, it would be better if recipes weren't hard-coded like this. Instead you can copy the item id from the 8th slot to the 7th slot:
```
data modify block ~ ~ ~ Items[{Slot:7b,Count:1b}].id set from block ~ ~ ~ Items[{Slot:8b}]
```

Reply:
Can't use `data modify`, as this module is made for 1.13 atm

Reply:
For the 1.13 version, I'll make `pulverizers:set_slot` run from a tag `#pulverizers:set_slot` to allow for expansions. And make the changes for the 1.14 version use `data modify`